### PR TITLE
Update change-log.md

### PR DESCRIPTION
**Pull request by MartaD**
product update for 6th Jan 2023

### CODE REVIEW

File: docs/change-log.md [hunk 1]
```diff
@@ -2,6 +2,18 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update January 6th, 2022
```
Comment (sujaya-sys) [inline docs/change-log.md:5]: 2023 :-D

File: docs/change-log.md [hunk 2]
```diff
@@ -2,6 +2,18 @@
...
+
+- **Improved:** Responsive design of [Application Details Screen](https://docs.humanitec.com/reference/user-interface/app-details-screen) has been improved.
+
+- **Fixed:** Workload variables with multiple [Placeholders](https://docs.humanitec.com/reference/concepts/app-config/placeholders) render correctly in the UI.
+
+- **Fixed:** [`Create new app`](https://docs.humanitec.com/getting-started/create-sample-app#create-an-app-in-humanitec) input field validation no longer expands beyond the base element.
+
+- **Fixed:** Alignment of the dropdown menu of Deployments on [Workload Details Screen](https://docs.humanitec.com/reference/user-interface/workload-details-screen).
```
Comment (sujaya-sys) [inline docs/change-log.md:13]: which ticket is this referring to? The dropdown items still appear buggy to me https://humanitec.atlassian.net/browse/WAL-5709

Comment (MartaD) [reply docs/change-log.md:13]: this one: https://humanitec.atlassian.net/browse/WAL-6025
but I've just noticed it wasn't worked on, probably a duplicate.. I'll remove this line

File: docs/change-log.md [hunk 3]
```diff
@@ -2,6 +2,21 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update January 6th, 2023
+
+- **Improved:** More intuitive and easier to read design for table components across the application:
+    - Shared App Values & Secrets on [App Settings Screen](https://docs.humanitec.com/guides/orchestrate-infrastructure/manage-environment-variables-and-secrets#app-level);
+    - App Values & Secrets Overrides on [Envrionment Settings Screen](https://docs.humanitec.com/guides/orchestrate-infrastructure/manage-environment-variables-and-secrets#environment-level);
+    - [Variables](https://docs.humanitec.com/using-humanitec/work-with-workloads/work-with-environment-variables), [Files](https://docs.humanitec.com/using-humanitec/work-with-workloads/work-with-files), and [Volume Mounts](https://docs.humanitec.com/guides/orchestrate-infrastructure/manage-persistent-volumes) on [Workload Details Screen](https://docs.humanitec.com/reference/user-interface/workload-details-screen).
```
Comment (sujaya-sys) [inline docs/change-log.md:10]: you're ending this list item with a `.` while the others end with a `;`

File: docs/change-log.md [hunk 4]
```diff
@@ -2,6 +2,21 @@
...
+
+- **Improved:** Responsive design of [Application Details Screen](https://docs.humanitec.com/reference/user-interface/app-details-screen) has been improved.
```
Comment (sujaya-sys) [inline docs/change-log.md:12]: I suggest to scratch "has been improved" at the end, as it reads like a duplication

File: docs/change-log.md (hunk 3 above)
Comment (MartaD) [reply docs/change-log.md:10]: wasn't sure how to punctuate the bullet list, it's ok that the middle ones are ended with semicolons and only the last with a dot, don't know what's the standard for us. Anyway, I've replaced semicolons with commas

**Review by sujaya-sys — APPROVED** [review]
LGTM

**Review by sujaya-sys — APPROVED** [review]
LGTM

**Pull request merged**